### PR TITLE
Tailsitter

### PR DESCRIPTION
**Pull request by romain-chiap**
Tailsitter airframe to be used with SIH!
PR-[18162](https://github.com/PX4/PX4-Autopilot/pull/18162)

I'm not able to have the correct color, the vehicle appears black or white (I did set shading to 0.0), @bkueng any idea?

The vehicle looks like that in Blender
Screen Shot 2021-08-27 at 5 52 32 PM

### CODE REVIEW

**Review by bkueng — COMMENTED** [review]
> I'm not able to have the correct color, the vehicle appears black or white (I did set shading to 0.0), @bkueng any idea?

I never looked into that for the existing model, not sure how it's done there.

File: models/x_vert.mtl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# newmtl material_black
+# Ns 96.078431
+# Ka 0.000000 0.000000 0.000000
+# Kd 0.053447 0.053447 0.053447
+# Ks 0.500000 0.500000 0.500000
+# Ke 0.000000 0.000000 0.000000
+# Ni 1.000000
+# d 1.000000
+# illum 2
+# map_Kd /Users/romain/Documents/projet_perso/technical_projects/tailsitter/map_black.png
```
Comment (bkueng) [inline models/x_vert.mtl:13]: This should not include your local path

**Review by bkueng — COMMENTED** [review]
@romain-chiap did you want to merge this as is?

**Review by romain-chiap — COMMENTED** [review]
we're not using these absolute paths, they were commented already

**Pull request merged**